### ISSUE TITLE
Add new line at the end of declaration/implementation for cleaner diffs

### ISSUE THREAD

**Issue by Roald87** [Barteling/STweep#2]
Currently if you add some code on the last line of (mainly) the implementation part and there is no new line at the end of the file, you will see the following diff:

```diff
-var1 := 1;]]></ST>
+var1 := 1;
+var2 := 2;]]></ST>
```

However, if there would be a new line at the end of the file you will see:

```diff
var1 := 1;
+var2 := 2;
]]></ST>
```

For the declaration part it will be less important, since they will usually end with `END_VAR`, but still a nice improvement for the few times it is needed.

**Comment by Barteling** [comment]
Do you not use the TwinCAT diff tool to compare objects? Then you don't have to worry about the surrounding XML.
Anyway I believe this is a small addition i'll put it on the list

**Comment by Roald87** [comment]
No not really. Somehow the TcCompare tool breaks all the time and I gave up using it. It was especially annoying if I wanted to compare for example xml files while TcCompare was selected as the compare tool. It refused to open the xml file and there was no easy way to switch to another tool. Also it is convenient if you view a commit on Gitlab/GitHub/etc., since they always show the plain text files.

**Comment by Barteling** [comment]
Fixed in 3.0.0-Beta
